config.mk

```diff
@@ -4,4 +4,7 @@ TARGET=ar71xx
 PACKAGES_LIST_DEFAULT=default backbone
 OPENWRT_SRC=git://git.openwrt.org/15.05/openwrt.git
 OPENWRT_COMMIT=64e116779c0f7da6d98068b8e7c50f528c8a91f2
+REPOPATH=http://buildbot.berlin.freifunk.net/buildbot/unstable/%T/928/packages/
```

Comment:
This was just an corresponding buildnumber to check that the setting is correctly applied.
The whole line gets removed finally.